components/form/index.css

```diff
@@ -46,7 +46,7 @@
 	display: block;
 }
 
-/* Fix extra space after inline-flex elements such as stepper. */
+/* Fix extra space after inline-flex elements such as number field. */
```

Comment:
Feel free to tell me to revert this- it's probably not necessary but I wanted to clarify the component's new name.

Reply:
Ohh I'd put it on my to-do list to look at why this component had changes. From a release standpoint, it seems ok? Once it's migrated, it'll be bumped to the next major version anyway.

Reply:
The only things I changed in form's CSS is this comment (and then the form stories, I had to update the import to `NumberField`). 

Is the comment change necessary right now- probably not. I was just trying to cover all the instances of the word "stepper" and update the language to "number field."

Think I should remove this comment?